src/test/java/net/starlark/java/eval/StarlarkIntTest.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package net.starlark.java.eval;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import static com.google.common.truth.Truth.assertThat;
+
+/** Tests for int. */
+@RunWith(JUnit4.class)
+public class StarlarkIntTest {
+
+  @Test
+  public void testUminus() {
+    assertThat(StarlarkInt.uminus(StarlarkInt.of(0))).isEqualTo(StarlarkInt.of(0));
```

Comment:
Please write this test in Starlark, in testdata/int.star.

(I plan to move the Java-based tests to Starlark in my copious free time.)